### ISSUE TITLE
Is there a way I can get Phonetic translation instead of Dictionary translations?

### ISSUE THREAD

**Issue by Abhi150688** [shvets/google-translate#20]
I have a very specific usecase where I need phonetic translations instead of dictionary ones. Is there a way, the api can be customized for the same?

Google translate does allow for phonetic translations as per article here: http://www.androidcentral.com/google-translate-app-gets-phonetic-support-more-languages

**Comment by guyrotem** [comment]
yes, you can (for some languages). when there is a phonetic translation, it will be available at response[0][0][1][2].
See the following example: "Cèshì" is the phonetic translation (en -> zh) of the word "test":

[[["测试","test",,,2],[,,"Cèshì","test"]],[["noun",["测试","试验","试","实验","考试","考验","测验"], ...
